specification/iothub/resource-manager/Microsoft.Devices/stable/2020-04-01/iothub.json

```diff
@@ -3360,6 +3361,25 @@
             "name": "EndpointHealthStatus",
             "modelAsString": true
           }
+        },
+        "lastknownerror": {
+          "description": "Last error obtained when a message failed to be delivered to iot hub",
+          "type": "string"
+        },
+        "lastKnownErrorTime": {
+          "description": "Timestamp of when the last known error ocurred",
+          "type": "string",
+          "format": "date-time-rfc1123"
+        },
+        "lastSuccessfulSendAttemptTime": {
+          "description": "Last time iot hub successfully tried to send a message to the endpoint",
```

Comment:
"successfully tried" or "successfully sent" ?

Reply:
sent let me fix it